linux57-tkg/install.sh

```diff
@@ -137,7 +142,14 @@ if [ "$1" = "install" ]; then
       export CCACHE_SLOPPINESS="file_macro,locale,time_macros"
       export CCACHE_NOHASHDIR="true"
       msg2 'ccache was found and will be used'
+
+    elif [ "$_distro" = "Fedora" ] && dnf list --installed ccache > /dev/null; then
+      export PATH="/usr/lib64/ccache/:$PATH"
+      export CCACHE_SLOPPINESS="file_macro,locale,time_macros"
+      export CCACHE_NOHASHDIR="true"
```

Comment:
Since this looks to be the same between distributions, this could be moved into the outer `if`.

Reply:
Will do, needs a small tweak, you tell me what you think of it. I will push the change in a few minutes